xla/service/gpu/transforms/async_wrapper_test.cc

```diff
@@ -81,5 +82,132 @@ TEST_F(AsyncWrapperTest, BasicFusion) {
   EXPECT_TRUE(LiteralTestUtil::Equal(expected, result));
 }
 
+TEST_F(AsyncWrapperTest, BasicFusionWithinWhile) {
```

Comment:
As indicated in other PRs, let's explain what the behavior being tests is in the test name. (Same below.)

Reply:
Done.